package-lock.json

```diff
@@ -5,9 +5,19 @@
   "requires": true,
   "dependencies": {
     "@ardatan/aggregate-error": {
-      "version": "0.0.1",
-      "resolved": "https://registry.npmjs.org/@ardatan/aggregate-error/-/aggregate-error-0.0.1.tgz",
-      "integrity": "sha512-UQ9BequOTIavs0pTHLMwQwKQF8tTV1oezY/H2O9chA+JNPFZSua55xpU5dPSjAU9/jLJ1VwU+HJuTVN8u7S6Fg=="
+      "version": "0.0.6",

```

Comment:
This seems like a very significant change. Is this necessary?

Reply:
I think this change is caused due to downgrading the version of Gatsby.

Reply:
Everything looks good @jomefavourite . You just don't have to commit the changes to this file and we are good to go

Reply:
@Tanuj22 do you mean I should revert the changes?

Reply:
Yes